app/views/layouts/mailer.html.erb

```diff
@@ -8,6 +8,6 @@
   </head>
 
   <body>
-    <%= yield %>
+    yield: <%= yield %>
```

Comment:
do you want this to say yield in the text?

Reply:
Saw this too after requesting review. I do not.

Reply:
It never shows anywhere though. But without the file, the app errors on submission.